cinder/autogenerated_client.go

```diff
@@ -0,0 +1,1821 @@
+package cinder
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+type RequestHandlerFn func(*http.Request) (*http.Response, error)
```

Comment:
It'd be good to document this, and describe that the auto-generated functions will have a hard-coded URL that needs to be overridden by the `RequestHandlerFn`. Maybe move it out of the auto-generated file as well.